### PR TITLE
export recommendations with extra labels

### PR DESCRIPTION
**Pull request by MaximilianKrambach**

### CODE REVIEW

File: client/src/app/site/motions/services/motion-csv-export.service.ts [hunk 1]
```diff
@@ -52,8 +57,7 @@ export class MotionCsvExportService {
                 { property: 'title' },
                 {
                     label: 'recommendation',
-                    map: motion =>
-                        motion.recommendation ? this.translate.instant(motion.recommendation.recommendation_label) : ''
+                    map: motion => (motion.recommendation ? this.motionRepo.getExtendedRecommendationLabel(motion) : '')
```
Comment (emanuelschuetze) [inline client/src/app/site/motions/services/motion-csv-export.service.ts]: this works, but it is for the call list only. Please add your fix also for the motion list export - for extended state and recommenation label.

**Comment by MaximilianKrambach** [comment]
should work now @emanuelschuetze

**Comment by emanuelschuetze** [comment]
CSV works good now. But the new XLSX export needs the same fix. Maybe @tsiegleauq can help you.

**Comment by MaximilianKrambach** [comment]
@tsiegleauq I think I adapted the xlsx export, too (see second commit), but I am unsure if the motionRepo is 'allowed' there.

**Pull request merged**